src/main/webapp/resources/celJS/initCelements.js

```diff
@@ -638,6 +644,214 @@
+        var _me = this;
+        _me._eventElements = new Array();
+        _me._interpretDataCelEventBind = _me._interpretDataCelEvent.bind(_me);
+        _me._contentChangedHandlerBind = _me._contentChangedHandler.bind(_me);
+        _me.updateCelEventHandlersBind = _me.updateCelEventHandlers.bind(_me);
+      },
+      
+      _splitDataCelEventList : function(dataValue) {
+        var _me = this;
+        var ret = new Array();
+        if (dataValue) {
+          ret = dataValue.split('&');
+        }
+        return ret;
+      },
+
+      _parseEventInstruction : function(instruction) {
+        var _me = this;
+        var parts = instruction.match(_me._instructionRegex) || [];
+        var data = {
+          eventName : parts[1],
+          action : parts[2],
+          className : parts[3],
+          cssSelector : parts[4],
+          condition : parts[5]
+        };
+        if (data.eventName === 'load') {
+          // action won't ever trigger on load, using earliest possible
+          data.eventName = 'cel:onEventInitialized';
+        }
+        if (data.eventName && data.action && data.className && data.cssSelector) {
+          return data;
+        } else {
+          throw "parseEventInstruction: unable to parse event instruction '" + instruction + "'"; 
+        }
+      },
+
+      _createEventHandler : function(htmlElem, instruction) {
+        var _me = this;
+        var data = _me._parseEventInstruction(instruction);
+        var actionFunction = _me._actionFunctionMap[data.action];
+        if (actionFunction) {
+          return new window.CELEMENTS.CssClassEventHandler(htmlElem, data.eventName,
+              data.cssSelector, data.className, actionFunction, data.condition);
+        } else {
+          throw "createEventHandler: unknown action '" + data.action + '"';
+        }
+      },
+
+      _interpretDataCelEvent : function(htmlElem) {
+        var _me = this;
+        var logPref = 'EventManager - interpretData: ';
+        if (!htmlElem.hasClassName('celOnEventInitialized')) {
+          var dataValue = htmlElem.readAttribute('data-cel-event');
+          var newElem = {
+              'htmlElem' : htmlElem,
+              'dataValue' : dataValue,
+              'eventHandlers' : new Array()
+          };
+          _me._splitDataCelEventList(dataValue).each(function(instruction) {
+            try {
+              newElem.eventHandlers.push(_me._createEventHandler(htmlElem, instruction));
+            } catch (exp) {
+              console.error(logPref, 'invalid instruction ', exp, htmlElem);
+            }
+          });
+          if (newElem.eventHandlers.length > 0) {
+            _me._eventElements.push(newElem);
+            console.debug(logPref, 'new element ', htmlElem);
+          } else {
+            console.debug(logPref, 'no valid instructions found on ', htmlElem);
+          }
+          htmlElem.addClassName('celOnEventInitialized');
+          htmlElem.fire('cel:onEventInitialized');
```

Comment:
please use shorter event-name for example:
```suggestion
          htmlElem.fire('celEM:init');
```